sprout/scripts/config_validation/bgcf_schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "title": "All BGCF routes",
+  "type": "object",
+  "properties": {
+    "routes": {
+      "type": "array",
+      "items": {
+        "title": "Each BGCF route",
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "domain": {
+            "type": "string",
+            "description": "SIP trunk IP address or domain name"
```

Comment:
It may not always start with www or https, or end with com or co.uk right? I can only tell that it should contain at least one dot